capture/run_doit.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
calling this file `tasks` or `jobs` might be more in line with a workflow manager / the doit vocabulary